src/Valleysoft.DockerCredsProvider/CredsProvider.cs

```diff
@@ -116,4 +117,21 @@ private static async Task<ICredStore> GetCredStoreAsync(string registry, IFileSy
 
         throw new CredsNotFoundException($"No matching auth specified for registry '{registry}' in Docker config.");
     }
+
+    private static string ConvertToHostname(string input)
+    {
+        string stripped = input;
+        if(input.StartsWith("http://")) 
```

Comment:
Please define a `const` for `http://` and `https://`.

Reply:
done.